src/sinks/util/sink.rs

```diff
@@ -643,16 +643,25 @@ where
         let response = self
             .service
             .call(req)
-            .map(move |response| {
-                trace!(message = "Response successful.", ?response);
-                (seqno, batch_size)
-            })
             .map_err(Into::into)
-            .then(|res| {
+            .then(move |result| {
+                match result {
+                    Ok(response) => {
+                        trace!(message = "Response successful.", ?response);
+                    }
+                    Err(error) => {
+                        error!(
```

Comment:
Might be worth rate limiting this?

Reply:
I agreed initially but after thinking more I'm going to leave it out. We generally rate limit event-level logs where there could potentially be hundreds of thousands per second. This is a request-level logs where the value is likely higher and we have default rate limits around how many we'll do in a given period of time (usually tens or hundreds per second at most).

That said, we can always revisit.